python/builtInExample2.py

```diff
@@ -0,0 +1,144 @@
+#### import the simple module from the paraview
+from paraview.simple import *
+
+# Load the scalar fields using 'XML Image Data Reader'
+example2vti = XMLImageDataReader(FileName=['BuiltInExample2.vti'])
+
+### Univariate data analysis using isosurfaces
+
+# create a new 'Contour' for 'log(Rho)' to identify atoms
+contour2 = Contour(Input=example2vti)
+contour2.ContourBy = ['POINTS', 'log(Rho)']
+contour2.Isosurfaces = [1.57]
+contour2.PointMergeMethod = 'Uniform Binning'
+
+# create a new 'Contour' for 'log(s)' to identify atoms
+contour3 = Contour(Input=example2vti)
+contour3.ContourBy = ['POINTS', 'log(s)']
+contour3.Isosurfaces = [-0.575]
+contour3.PointMergeMethod = 'Uniform Binning'
+
+### Bivariate data analysis using isosurfaces
+
+# create a new 'TTK ContinuousScatterPlot'
+tTKContinuousScatterPlot1 = TTKContinuousScatterPlot(Input=example2vti)
+tTKContinuousScatterPlot1.ScalarField1 = ['POINTS', 'log(Rho)']
+tTKContinuousScatterPlot1.ScalarField2 = ['POINTS', 'log(s)']
+
+# create a new 'Threshold'
+threshold1 = Threshold(Input=tTKContinuousScatterPlot1)
+threshold1.Scalars = ['POINTS', 'ValidPointMask']
+threshold1.ThresholdRange = [0.1, 1.0]
+
+# create a new 'TTK ProjectionFromField' to project the 'log(Rho)' contour onto the range space
+tTKProjectionFromField1 = TTKProjectionFromField(Input=contour2)
+tTKProjectionFromField1.UComponent = ['POINTS', 'log(Rho)']
+tTKProjectionFromField1.VComponent = ['POINTS', 'log(s)']
+
+# create a new 'Extract Edges'
+extractEdges1 = ExtractEdges(Input=tTKProjectionFromField1)
+
+# create a new 'Tube'
+tube1 = Tube(Input=extractEdges1)
+tube1.Scalars = ['POINTS', 'log(Rho)']
+tube1.Vectors = ['POINTS', 'Normals']
+tube1.Radius = 0.02
+
+# create a new 'TTK ProjectionFromField' to project the 'log(s)' contour onto the range space
+tTKProjectionFromField2 = TTKProjectionFromField(Input=contour3)
+tTKProjectionFromField2.UComponent = ['POINTS', 'log(Rho)']
+tTKProjectionFromField2.VComponent = ['POINTS', 'log(s)']
+
+# create a new 'Extract Edges'
+extractEdges2 = ExtractEdges(Input=tTKProjectionFromField2)
+
+# create a new 'Tube'
+tube2 = Tube(Input=extractEdges2)
```

Comment:
I would suggest no to create/store graphical primitives here (see my comment below).